.editorconfig

```diff
@@ -17,3 +17,6 @@ charset = utf-8-bom
 
 [*.md]
 trim_trailing_whitespace = false
+
+[*.snap]
+trim_trailing_whitespace = false
```

Comment:
Snapshots can have trailing whitespace, you can have multiple imports on the same line (and have spaces between them), so after processing we have trailing whitespace (because original import was removed to replace on generated code and spaces between imports just kept), so let's avoid problems with IDE here